pytorch_toolkit/object_detection/tests/export_tests_face_detection.py

```diff
@@ -12,42 +12,70 @@
 # See the License for the specific language governing permissions
 # and limitations under the License.
 
-from common.test_case import export_test_case
-from common.utils import replace_text_in_file
+from common.test_case import create_export_test_case
 
 
-def face_detection_test_case(model_name, alt_ssd_export):
-    class ExportTestCase(export_test_case('face-detection', model_name, alt_ssd_export=alt_ssd_export)):
-        def setUp(self):
-            super().setUp()
-
-            assert replace_text_in_file(self.configuration_file, 'data/WIDERFace',
-                                        '../../data/airport')
-            assert replace_text_in_file(self.configuration_file, 'val.json',
-                                        'annotation_faces_train.json')
-
-    return ExportTestCase
-
-
-class FaceDetection0200TestCase(face_detection_test_case('face-detection-0200', True)):
+class FaceDetection0200TestCase(
+        create_export_test_case(
+            'face-detection',
+            'face-detection-0200',
```

Comment:
I have a proposal here. What if we will not pass ```problem name``` which is pretty similar to ```model name``` and pass only the second parameter? It will require the line like 
`problem_name='-'.join(model_name.split('-')[:-1])
` 
in the  ```setUpClass``` method in the ```create_export_test_case``` function. 
On the one hand, it will reduce duplications. But from the other, it will make code more confusing, I suppose.

Reply:
At least my proposal here is to use named args like `problem='face-detection', model='face-detection-200'`

Reply:
I leave it as is. I did not find your proposals significant enough.